src/Ecommerce/Cart/PriceModificator/Shipping.php

```diff
@@ -67,7 +68,7 @@ public function getName()
      *
      * @return ModificatedPriceInterface
      */
-    public function modify(PriceInterface $currentSubTotal, CartInterface $cart)
+    public function modify(PriceInterface $currentSubTotal, CartInterface $cart): ModificatedPrice|ModificatedPriceInterface
```

Comment:
why not just return `ModificatedPriceInterface`?

Reply:
parent class has same signature.

Reply:
ModificatedPrice implements ModificatedPriceInterface. So ModificatedPriceInterface is enough